inc/core/application.hpp

```diff
@@ -32,6 +32,11 @@
 
 namespace gearshifft {
 
+  inline
+  std::string gearshifft_version() {
```

Comment:
why the redundancy? why not have `gearshifft::version()` instead of `gearshifft::gearshifft_version()`?

Reply:
yes, this is redundant, thanks for spotting, I gonna remove it in this PR.